bundles/org.openhab.ui/web/src/components/widgets/system/oh-sipclient.vue

```diff
@@ -0,0 +1,248 @@
+<template>
+  <div>
+    <!-- Show settings gear for local settings if intercom is enabled and in edit mode -->
+    <f7-button v-if="context.editmode" :style="{ height: config.iconSize + 'px' }" icon-f7="gear_fill" :icon-size="config.iconSize + 'px'" @click.stop="localSettingsPopup()" />
+    <div v-if="config.enableVideo" class="video-container">
+      <video ref="remoteVideo" autoplay playsinline class="remote-video" poster="@/images/openhab-logo.svg" />
+      <video v-if="config.enableLocalVideo" ref="localVideo" autoplay playsinline muted="muted" class="local-video" />
+    </div>
+    <div v-if="!context.editmode">
```

Comment:
It's nice to have the whole widget visible when editing so you have a sense of how it looks and is sized, i would recommend reverting this so it shows the yellow "not connected" icon as a placeholder, especially since an icon is always present when it's running.    I have another PR with a few changes i will open up shortly, i can can change this there if we agree.

Reply:
You’ve definitely got a good point,  so let’s display the placeholder on edit mode.

Reply:
I've seen that you already did that in your last PR, which I merged a few minutes ago. Thanks :+1: